packages/studioCMS/src/integrations/studioCMSDashboard/routes/authroutes/login/index.astro

```diff
@@ -48,31 +49,23 @@ if (locals.isLoggedIn) {
 
           <form method="post" action={loginAPI}>
 
-              <Input 
-                name="username" 
-                id="username" 
-                label='Username'
-                /><br />
-
-              <Input 
-                type="password" 
-                name="password"
-                id="password"
-                label="Password"
-                password-toggle
-                /><br />
-
-            <center>
+              <Input name="username" id="username" label='Username' />
+
+              <Divider />
+
+              <Input type="password" name="password" id="password" label="Password" password-toggle />
+
+              <Divider />
+
+            <div style="text-align:center">
               <Button variant="primary" type="submit" outline>Continue <Icon slot="suffix" library="mdi" name="login"/></Button>
-            </center>
+            </div>
 
           </form>
 
           {allowUserRegistration && (
-            <div class="py-4">
-              <center>
+            <div class="py-4" style="text-align:center">
```

Comment:
`style` attribute again???

Reply:
I don't know why, but with specifically the Shoelace buttons this is the easiest way to center them without using a `<center>` tag

Reply:
never mind... issue resolved itself... swapped in [27a6991](https://github.com/astrolicious/studiocms/pull/156/commits/27a6991667da335a1af5a8fabae7f28eb59d771e)